testsuite/omsimulator/DualMassOscillator_me.mos

```diff
@@ -64,7 +64,8 @@ val(system2.s2, {0.0,0.1});
 // true
 // ""
 // 0
-// "info:    maximum step size for 'DualMassOscillator.root': 0.100000
+// "warning: Alg. loop (size 4)
```

Comment:
Where is this warning coming from? Is it from `listAllIterationVariables0` ?
And the warning is very cryptic. Warning, you have a loop, what am I supposed to do with this information?

Reply:
It is coming from OMSimulator and should indeed be improved. But that is out of the scope of this PR.

Reply:
But what changed that OMSimulator now prompts this warning and not before?

Reply:
This PR fixes the section `ModelVariables` in the `modelDescription.xml` file, i.e. variability, causality, initial, and start attribute. The importing tool, i.e. OMSimulator, has to derive initial unknowns from the `ModelVariables`, because the OpenModelica FMI export doesn't generate them yet. Due to the changes from this PR OMSimulator is now able to detect circular dependencies between the FMUs.